mycroft/client/speech/listener.py

```diff
@@ -141,15 +142,17 @@ def transcribe(self, audio):
             LOG.error("Could not request Speech Recognition {0}".format(e))
         except ConnectionError as e:
             LOG.error("Connection Error: {0}".format(e))
-            self.__speak("Mycroft seems not to be connected to the Internet")
+            self.__speak(mycroft.dialog.get("not connected to the internet",
+                                            self.stt.lang))
         except HTTPError as e:
             if e.response.status_code == 401:
                 text = "pair my device"
                 LOG.warn("Access Denied at mycroft.ai")
         except Exception as e:
             LOG.error(e)
             LOG.error("Speech Recognition could not understand audio")
-            self.__speak("Sorry, I didn't catch that")
+            self.__speak(mycroft.dialog.get("i didn't catch that",
+                                            self.stt.lang))
```

Comment:
Yeah, I've been fighting this for a while... the official lang codes are mixed case.  But all the Skills in the world already use lowercase in the file structure.  I think using the lang.lower() within the dialog.get() is probably an acceptable convention.  I can't imagine anyone wanting to distinguish "en-US" from "en-us" or "en-Us".